collisions_functions.py

```diff
@@ -0,0 +1,197 @@
+"""This module defines atom class and functions used in an elastic
+collision physics simulation.
+"""
+
+import math
+import random
+import numpy
+from random import randrange, uniform
+
+
+# Parameters
+red = (255,0,0)
+
+
+class Atom:
+    """ Defines an atom structure. """
+    def __init__(self, rad, color, max_vel, boundary):
+        self.ID = -1

```

Comment:
Can you help me understand what the `ID` field is used for?

Reply:
Apparently nothing now! I had been using that to indicate which atoms were colliding but forgot to remove it when I reorganized that.